accord-core/src/main/java/accord/local/SaveStatus.java

```diff
@@ -74,6 +74,18 @@ public boolean hasBeen(Status status)
         return this.status.compareTo(status) >= 0;
     }
 
+    public boolean isComplete()
+    {
+        switch (this)
+        {
+            case Applied:
+            case Invalidated:
+                return true;
+            default:
+                return false;
```

Comment:
I would prefer not to use `default` properties, as it makes it harder to safely modify the state machine in future.

Reply:
Is this actually used?

Reply:
it was, but then deleted in favor of something else; but forgot to purge this